spec/fixtures/constants.rb

```diff
@@ -575,3 +575,17 @@
                          %d
 1 warning generated.
 )
+
+
+################################################################################
+# INFORMATIVE
+################################################################################
+
+SAMPLE_PLUGIN_INFO = %Q(2015-04-23 10:20:09.018 xcodebuild[77828:9774464] [MT] PluginLoading: Required plug-in compatibility UUID 9F75337B-21B4-4ADC-B558-F9CADF7073A7 for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/AllMissingXcodeFeatures.xcplugin' not present in DVTPlugInCompatibilityUUIDs)
+
+SAMPLE_PLUGIN_INFO_TRIMMED = %Q([MT] PluginLoading: Required plug-in compatibility UUID 9F75337B-21B4-4ADC-B558-F9CADF7073A7 for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/AllMissingXcodeFeatures.xcplugin' not present in DVTPlugInCompatibilityUUIDs)
+
+SAMPLE_IPHONE_SIMULATOR_INFO = %Q(2015-04-23 10:20:09.161 xcodebuild[77828:9774464] [MT] iPhoneSimulator: SimVerifier returned: Error Domain=NSPOSIXErrorDomain Code=53 "Simulator verification failed." UserInfo=0x7f81829c2240 {NSLocalizedFailureReason=A connection to the simulator verification service could not be established., NSLocalizedRecoverySuggestion=Ensure that Xcode.app is installed on a volume with ownership enabled., NSLocalizedDescription=Simulator verification failed.})
```

Comment:
Line is too long. [486/80]<br>Use `%` instead of `%Q`.<br>Do not use `%Q` unless interpolation is needed.  Use `%q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.